web/packages/hovercards/src/core.ts

```diff
@@ -295,32 +366,229 @@ export default class Hovercards {
+				<div class="gravatar-hovercard__drawer-backdrop backdrop"></div>
+				<div class="gravatar-hovercard__drawer-card">
+					<div class="gravatar-hovercard__drawer-header">
+						<h2 class="gravatar-hovercard__drawer-title">${ titleText }</h2>
+						<button class="gravatar-hovercard__drawer-close close-btn">
+							<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+								<path d="M12 13.0607L15.7123 16.773L16.773 15.7123L13.0607 12L16.773 8.28772L15.7123 7.22706L12 10.9394L8.28771 7.22705L7.22705 8.28771L10.9394 12L7.22706 15.7123L8.28772 16.773L12 13.0607Z" fill="#101517"></path>
+							</svg>
+						</button>
+					</div>
+					<ul class="gravatar-hovercard__drawer-items">
+						${ content }
+					</ul>
+				</div>
+			</div>
+		`;
+	}
+
+	/**
+	 * Opens a hovercard drawer.
+	 *
+	 * @param {string}  selector  - The drawer selector.
+	 * @param {Element} container - The container context to search for the drawer.
+	 * @return {void}
+	 */
+	private static _openDrawer( selector: string, container: HTMLElement ): void {
+		const drawer = container.querySelector( selector );
+
+		if ( ! drawer ) {
+			return;
+		}
+
+		drawer.classList.add( 'gravatar-hovercard__drawer--open' );
+	}
+
+	/**
+	 * Closes a hovercard drawer.
+	 *
+	 * @param {string}  selector  - The drawer selector.
+	 * @param {Element} container - The container context to search for the drawer.
+	 * @return {void}
+	 */
+	private static _closeDrawer( selector: string, container: Element ): void {
+		const drawer = container.querySelector( selector );
+
+		if ( ! drawer ) {
+			return;
+		}
+
+		drawer.classList.add( 'gravatar-hovercard__drawer--closing' );
+		drawer.classList.remove( 'gravatar-hovercard__drawer--open' );
+
+		setTimeout( () => {
+			drawer.classList.remove( 'gravatar-hovercard__drawer--closing' );
+		}, 300 );
+	}
+
+	/**
+	 * Creates the contact drawer content.
+	 *
+	 * @param {Record< string, any >[]} contactsData - The user's contact data.
+	 * @return {string}                              - The contact drawer content.
+	 */
+	private static _createContactDrawerContent( contactsData: Record< string, any >[] ): string {
+		const icons: Record< string, string > = {
+			email: 'icons/mail.svg',
+			home_phone: 'icons/home-phone.svg',
+			work_phone: 'icons/work-phone.svg',
+			cell_phone: 'icons/mobile-phone.svg',
+			contact_form: 'icons/envelope.svg',
+			calendar: 'icons/calendar.svg',
+		};
+
+		const getUrl = ( type: string, value: string ) => {
+			switch ( type ) {
+				case 'email':
+					return `mailto:${ value }`;
+				case 'contact_form':
+				case 'calendar':
+					return value.startsWith( 'http' ) ? value : `https://${ value }`;
+				default:
+					return null;
+			}
+		};
+
+		const items = contactsData.map( ( [ key, value ]: string[] ) => {
+			const url = getUrl( key, value );
+			let text = value.replace( /^(https?:\/\/)/, '' );
+			text = text.replace( /^(www\.)/, '' );
```

Comment:
It could be a single `replace` to reduce the bundle size, if you want:

```suggestion
			const text = value.replace( /^(https?:\/\/)?(www\.)?/, '' );
```